setup.py

```diff
@@ -24,21 +24,21 @@
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Framework :: Django",
-        "Framework :: Django :: 3.2",
-        "Framework :: Django :: 4.1",
         "Framework :: Django :: 4.2",
         "Framework :: Django :: 5.0",
+        "Framework :: Django :: 5.1",
+        "Framework :: Django :: 5.2",
```

Comment:
idt the pyproject.toml PR includes these changes

Reply:
Yeah, rebased this and applied the changes to `pyproject.toml` instead 👍